.circleci/config.yml

```diff
@@ -106,17 +144,22 @@ jobs:
 
 ## WORKFLOWS ##
 
-matrix: &matrix
+build_matrix: &build_matrix
+  matrix:
+    parameters:
+      os: [ alpine, debian, macos, "win/default" ]
+
+test_matrix: &test_matrix
   matrix:
     parameters:
-      os: [ alpine, debian ]
+      os: [ alpine, debian, macos ]
 
 workflows:
   postject:
     jobs:
       - build:
-          <<: *matrix
+          <<: *build_matrix
```

Comment:
I can go either way - the `test_matrix` should be temporary until we run tests on Windows, so I think the current changes provide the least amount of jank. Once tests are running on Windows we'll just drop `test_matrix`, rather than needing to un-inline it.